server/BraveAlerterConfigurator.js

```diff
@@ -12,12 +12,7 @@ class BraveAlerterConfigurator {
     return new BraveAlerter(
       this.getAlertSession.bind(this),
       this.getAlertSessionByPhoneNumbers.bind(this),
-      this.getAlertSessionBySessionIdAndAlertApiKey.bind(this),
       this.alertSessionChangedCallback.bind(this),
-      this.getLocationByAlertApiKey.bind(this),
-      this.getActiveAlertsByAlertApiKey.bind(this),
-      this.getHistoricAlertsByAlertApiKey.bind(this),
-      this.getNewNotificationsCountByAlertApiKey.bind(this),
```

Comment:
Nice. This change simplifies brave-alert-lib a lot, eh?

Reply:
So much!!